manim/mobject/svg/text_mobject.py

```diff
@@ -149,6 +157,9 @@ def __init__(self, text, **config):
         if self.height is None and self.width is None:
             self.scale(TEXT_MOB_SCALE_FACTOR)
 
+    def __repr__(self):
+        return f"Text({repr(self.original_text)})"
```

Comment:
Question: why does this one hardcode `Text` while the previous one does `type(self).__name__`?

Reply:
I was lazy and only wrote one `__repr__` for the entire Tex class structure, while `Text` does not have subclasses (any more, at least). This is why I thought using `type(self).__name__` might be overkill.